release.yaml

```diff
@@ -25,9 +25,9 @@ rancher-backup:
 rancher-backup-crd:
 - 2.1.5-rc2
 rancher-cis-benchmark:
-- 2.1.1-rc7
+- 2.1.1-rc8
 rancher-cis-benchmark-crd:
-- 2.1.1-rc7
+- 2.1.1-rc8
```

Comment:
@vardhaman22 Can we add the next chart version to merge this fix in, now that v2.6.11 has released?

Reply:
done @prachidamle